.ci/opensearch/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3'
+
+services:
+
+  opensearch:
+    build:
+      context: .
```

Comment:
Curious what does this signify ?

Reply:
The context is for a path to a directory containing a Dockerfile https://docs.docker.com/compose/compose-file/compose-file-v3/#build